package.json

```diff
@@ -181,6 +181,7 @@
     ]
   },
   "engines": {
-    "node": "18.x"
-  }
-}
+    "node": "20.x"
```

Comment:
according to https://docs.npmjs.com/cli/v10/configuring-npm/package-json#engines
it is better to make it `>=20.5.0`
ref: https://github.com/npm/cli/blob/latest/package.json#L264

Reply:
vercel appears to be fine with any format https://vercel.com/docs/functions/runtimes/node-js#version-overrides-in-package.json
however netlify still relies on .nvmrc for node verison

Reply:
nvmrc does support version range as per this:

https://www.squash.io/how-to-write-an-nvmrc-file-for-automatic-node-version-change/

<img width="1012" alt="image" src="https://github.com/coronasafe/care_fe/assets/25143503/b7af7b27-dbb5-4c3a-a08b-3d3daa04ea7e">

------

But an issue comment in nvm says otherwise... https://github.com/nvm-sh/nvm/issues/3167#issuecomment-1685022647

Reply:
We can set `nvmrc` to v`20.5.0` locked right?

Reply:
we can use [.node-version](https://github.com/shadowspawn/node-version-usage) 
both fnm and netlify support it echo `"20" > .node-version` should be enough as we won't have major breaking issues in 20.x